src/main/java/com/google/codeu/data/Message.java

```diff
@@ -25,18 +25,20 @@
   private String user;
   private String text;
   private long timestamp;
-  private String recipient; // Added recipient
+  private String recipient;  /** {@code recipient} */
```

Comment:
private String recipient; 

No need for comments for this :)

Reply:
Hi Emily! I just deleted the extra comment! May I merge the code to the master branch?

Reply:
yep!